sui_core/src/authority/authority_store.rs

```diff
@@ -608,9 +535,6 @@ impl<const ALL_OBJ_VER: bool, S: Eq + Serialize + for<'de> Deserialize<'de>>
     ) -> Result<(), SuiError> {
         let (objects, active_inputs, written, deleted, _events) = temporary_store.into_inner();
 
-        // Archive the old lock.
-        write_batch = write_batch.delete_batch(&self.transaction_lock, active_inputs.iter())?;
```

Comment:
So we no longer archive old locks?

Reply:
The old locks are deleted later ... look at the end of the method, there is a call to `self.lock_service.remove_locks()`... btw they aren't really "archived", the earlier code was also deleting the entries (not saved anywhere else).